core/templates/domain/collection/collection-playthrough.model.spec.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { CollectionPlaythrough } from './collection-playthrough.model';
+
+/**
+ * @fileoverview Tests for CollectionPlaythroughModel.
+ */
+
+describe('Collection playthrough model', () => {
+  it('should create a new CollectionPlaythrough instance', () => {
+    let collectionPlaythroughObject = new CollectionPlaythrough(
```

Comment:
Prefer `const` to `let` here since `collectionPlaythroughObject` is not re-assigned.

Reply:
Done

Reply:
Hmm this doesn't seem like this changed? I'm still seeing `let` here.

Reply:
Done